### DISCUSSION TITLE
RN vs Flutter

### DISCUSSION THREAD

**Discussion by muralitharanaspritle** [muralitharanaspritle/Event-GitHub-Day#13] [Polls]
Please answer from your heart

**Comment by muralitharanaspritle** [comment]
React Native

**Comment by MehediHasannOvi** [comment]
Flutter is a good choice if you want to develop an app that requires a high level of customization, while React Native may be a better choice if you need to integrate with existing JavaScript code or if you want to take advantage of the larger community and ecosystem. Ultimately, the choice between Flutter and React Native depends on the specific requirements and constraints of the project.
